app/main.py

```diff
@@ -1,8 +1,59 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
+    def __repr__(self) -> str:
+        return (f"Car(brand={self.brand}, "
+                f"clean_mark={self.clean_mark}, "
+                f"comfort_class={self.comfort_class})")
+
+
+bmw = Car(comfort_class=3, clean_mark=3, brand="BMW")
+audi = Car(comfort_class=4, clean_mark=9, brand="Audi")
+mercedes = Car(comfort_class=7, clean_mark=5, brand="Mercedes")
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_income = 0
+        for car in cars:
+            income_from_car = self.wash_single_car(car)
+            print(f"Washing {car.brand}: income = {income_from_car}")
```

Comment:
Consider removing or redirecting print statements: Printing directly within methods like `serve_cars` might not be suitable for production code. Consider using logging or returning the information instead.